specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1401,33 +1402,33 @@
             "name": "TokenSentimentValue",
             "modelAsString": false
           },
-          "description": "Opinion level sentiment for the aspect in the sentence."
+          "description": "Assessment sentiment in the sentence."
         },
         "confidenceScores": {
-          "description": "Opinion level sentiment confidence scores for the aspect in the sentence.",
-          "$ref": "#/definitions/AspectConfidenceScoreLabel"
+          "description": "Assessment sentiment confidence scores in the sentence.",
+          "$ref": "#/definitions/TargetConfidenceScoreLabel"
         },
         "offset": {
           "type": "integer",
           "format": "int32",
-          "description": "The opinion offset from the start of the sentence."
+          "description": "The assessment offset from the start of the sentence."
         },
         "length": {
           "type": "integer",
           "format": "int32",
-          "description": "The length of the opinion."
+          "description": "The length of the assessment."
         },
         "text": {
           "type": "string",
-          "description": "The aspect text detected."
+          "description": "The target text detected."
```

Comment:
Should this be "The assessment text detected." ?

Reply:
Thank u for catching this @ctufts :)